src/celengine/shadermanager.cpp

```diff
@@ -477,6 +479,7 @@ Sh_Expression operator*(const Sh_Expression& left, const Sh_Expression& right)
     return makeExpression<Sh_MultiplicationExpression>(left, right);
 }
 
+[[maybe_unused]]
```

Comment:
Leaving this in the codebase for completeness with the other arithmetic operators